src/Api/Billing/Public/Controllers/OrganizationsController.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System.Net;
+using Bit.Api.Models.Public.Response;
+using Bit.Api.Models.Request.Organizations;
+using Bit.Core.Context;
+using Bit.Core.Exceptions;
+using Bit.Core.OrganizationFeatures.OrganizationSubscriptions.Interface;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+using Bit.Core.Utilities;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+using OrganizationSubscriptionUpdateRequestModel = Bit.Api.Billing.Public.Models.OrganizationSubscriptionUpdateRequestModel;
+
+namespace Bit.Api.Billing.Public.Controllers;
+
+[Route("public/organizations")]
+[Authorize("Organization")]
```

Comment:
Seconded. Is there any reason why the existing controller can't be used?

Reply:
I thinking since activites in https://github.com/bitwarden/server/blob/master/src/Api/AdminConsole/Public/Controllers/OrganizationController.cs relates to adminConsole, We can change then new route to [Route("public/billing-organization")] or [Route("public/organization")] and still have it in Billing/Public/Controllers/OrganizationController.cs

Reply:
@withinfocus the existing controller is owned by AC Team, this is owned by Billing Team. Separate files for separate code ownership. But from a public api perspective, they both belong together on an `organizations/{id}/` endpoint.